db/migrate/20250312190444_set_listed_to_false_for_fetch_bot.rb

```diff
@@ -0,0 +1,9 @@
+class SetListedToFalseForFetchBot < ActiveRecord::Migration[6.1]
+  def change
+    # Fetch should not be marked as "listed", so it does not appear on the UI
+    BotUser.where(login: ['fetch']).each do |bot|
```

Comment:
It didn't need to be an array, could be just `login: 'fetch'`.

Reply:
Very true. I just removed the old items, and didn't update accordingly.